test/validation/api/ipsec/ipsec_test_out.c

```diff
@@ -407,22 +419,44 @@ static void test_out_in_common(ipsec_test_flags *flags,
 	    (auth == ODP_AUTH_ALG_NULL))
 		return;
 
+	if (flags->v4_tunnel) {
+		memset(&tunnel, 0, sizeof(odp_ipsec_tunnel_param_t));
```

Comment:
This leaves the TTL field zero which is bad. And same for hop limit in the IPv6 case.

Reply:
Sorry, hop limit was set, just TTL was not.

Reply:
Will add TTL. Will set it 64 like in other instances. Thanks for pointing out.